### PR TITLE
Add groupByMode=sql and responseFormat=sql options to query console

### PR DESCRIPTION
**Pull request by npawar**
Screen Shot 2019-10-09 at 5 48 23 PM

### CODE REVIEW

**Comment by kishoreg** [comment]
Nice!. Enable word seems redundant. It can be part of the tool tip if needed.
Group By Mode: Sql/sql Pql on by default.
Query Response Format: sql pql. pql on by default

**Comment by npawar** [comment]
> Nice!. Enable word seems redundant. It can be part of the tool tip if needed.
> Group By Mode: Sql/sql Pql on by default.
> Query Response Format: sql pql. pql on by default

Reworded and added tooltip. Are you suggesting radio buttons or toggle button?

File: pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PqlQueryResource.java [hunk 1]
```diff
@@ -76,8 +76,12 @@ public String post(String requestJsonStr, @Context HttpHeaders httpHeaders) {
       if (requestJson.has("trace")) {
         traceEnabled = requestJson.get("trace").toString();
       }
+      String queryOptions = null;
+      if (requestJson.has("queryOptions")) {
+        queryOptions = requestJson.get("queryOptions").asText();
+      }
       LOGGER.debug("Trace: {}, Running query: {}", traceEnabled, pqlQuery);
-      return getQueryResponse(pqlQuery, traceEnabled, httpHeaders);
+      return getQueryResponse(pqlQuery, traceEnabled, queryOptions, httpHeaders);
```
Comment (kishoreg) [inline pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PqlQueryResource.java]: traceEnabled can also be part of queryOptions rt

Comment (npawar) [reply pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PqlQueryResource.java]: If we move trace to queryOptions, the callers of the broker API will have to be changed to pass trace in queryOptions.

**Comment by npawar** [comment]
https://github.com/apache/incubator-pinot/issues/4450

**Pull request merged**